lms/djangoapps/instructor/permissions.py

```diff
@@ -132,5 +133,13 @@
 perms[MANAGE_COHORTS] = (
     HasAccessRule('instructor') |
     HasAccessRule('staff') |
-    HasPermissionRule(CourseRolesPermission.MANAGE_COHORTS)
+    HasPermissionRule(CourseRolesPermission.MANAGE_COHORTS) |
+    HasPermissionRule(CourseRolesPermission.MANAGE_STUDENTS)
```

Comment:
I assume that the addition of `MANAGE_STUDENTS` is intentional because that seems like an encompassing role, but is it possible to have it so that applying the `MANAGE_STUDENTS` role automatically applies the manage_cohorts role? or is that more complex?

Reply:
@connorhaugh Please let me know if my answer indicates I misunderstood the question. 

Adding the `course_roles` `MANAGE_STUDENTS` permission check in order to grant access to the instructor dashboard `MANAGE_COHORTS` permission is intentional. 

It is not possible to automatically apply a second permission when a first permission exists because apart from the permission itself the code will not know that the user should have that permission. Some `course_roles` roles will have both `MANAGE_STUDENTS` and `MANAGE_COHORTS`, but others may have only one or the other.